autopush/tests/test_rs_integration.py

```diff
@@ -81,21 +113,38 @@ class TestRustWebPush(unittest.TestCase):
     )
 
     def setUp(self):
```

Comment:
to avoid some pytest issues, might want to rename this to `setup_class` 

see https://docs.pytest.org/en/2.7.3/xunit_setup.html#class-level-setup-teardown and http://nose.readthedocs.io/en/latest/writing_tests.html

Reply:
Hmm, we're using setUp everywhere though?

Reply:
FYI this is a twisted trial TestCase, which is calling the setUp, regardless of test runner